app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -52,20 +47,23 @@ class MainFragment : Fragment() {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
 
-        viewModel.list.observe(this, Observer {
-            adapter = CurrentListAdapter(viewModel.list.value!!)
+        var recyclerView = view!!.findViewById<RecyclerView>(R.id.rec_currentList)
+        var userName = view!!.findViewById<TextView>(R.id.username)
```

Comment:
As I moved all the database logic to the MainViewModel, this is where the user-name and user-points - will be assigned to different views.